ruby/README.md

```diff
@@ -51,7 +51,9 @@ The runner also comes with a tool to investigate leaky tests:
 minitest-queue --queue path/to/test_order.log --failing-test 'SomeTest#test_something' bisect -Itest test/**/*_test.rb
 ```
 
-### RSpec
+### RSpec [DEPRECATED]
+
+> The rspec-queue runner is deprecated. The minitest-queue runner continues to be supported and is actively being improved. We recommend that new projects to set up their test suite using Minitest.
```

Comment:
Ugh, you copied my typo from the old version before I edited it.

Reply:
```suggestion
The rspec-queue runner is deprecated. The minitest-queue runner continues to be supported and is actively being improved. At Shopify, we strongly recommend that new projects set up their test suite using Minitest rather than RSpec.
```